maintenance_equipment_purchase/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Creu Blanca
```

Comment:
The module should be maintenance_request_purchase,
maintenance_equipment_purchase should be used for purchase of equipments :smile: